lib/utils.bash

```diff
@@ -263,7 +263,11 @@ find_install_path() {
     # And then use the binaries there
     local install_type="path"
     local version="path"
-    printf "%s\n" "${version_info[1]}"
+
+    util_resolve_user_path "${version_info[1]}"
+    local full_version="${REPLY}"
+
+    printf "%s\n" "$full_version"
```

Comment:
Why doesn't `util_resolve_user_path` print the resolved path directly? If it did we could simplify this to:

```bash
printf "%s\n" "$(util_resolve_user_path "${version_info[1]}")"
```

Reply:
@hyperupcall based on your latest comment, could this be changed to:

```bash
util_resolve_user_path "${version_info[1]}"
printf "%s\n" "${REPLY}"
```

I'd also prefer the REPLY variable be named something more specific, so as to better indicate it's origin. In this case, what about `$util_resolve_user_path_reply`?

```bash
util_resolve_user_path "${version_info[1]}"
printf "%s\n" "${util_resolve_user_path_reply}"
```